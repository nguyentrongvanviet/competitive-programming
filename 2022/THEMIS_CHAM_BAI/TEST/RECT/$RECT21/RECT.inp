162
-450432249 -712429621
-236344100 -624825104
791227431 -715143252
-462582738 -646329181
-240490397 220528299
947020307 -261672543
450224472 -288300554
509091618 -161211272
-143528567 -301308923
-34824795 -842788423
415032618 -32055621
-108903742 -264649594
787007449 -450097565
-31574595 901007912
825822282 -787917749
287544502 456703627
718981095 323365478
450224472 -288300554
-587967123 897955937
-400249728 -942677087
-674854311 -476573448
-647209859 672889159
-587967123 897955937
188035751 -244612057
-552581329 851808584
-534908200 -290248042
-167861958 499521574
853606040 889447908
300189459 -266788160
815199392 217662841
-729827642 -69182659
-363481761 159393710
-998025485 -784114349
-271807781 -572599777
932249198 788660962
525808958 -141817275
-601713556 -595197423
-849382962 -848721529
9683317 136357002
613494438 429015948
620499894 639622247
-720774233 30026782
-910072613 27981992
423587642 -357757886
-368490964 761436697
247617794 827051029
-998025485 -784114349
-413758431 -677044533
-497746302 814469485
603649420 621392588
-493590603 -152283105
823753163 275425263
635145413 -644370314
-356439891 -631718530
646310063 224483209
-395383253 -592026441
-802355754 -60569
708770388 43975432
287544502 456703627
995388592 -500065578
-446546734 94697233
164319316 -893772755
-244376073 -124288893
700041186 940894243
-970078100 444121410
955550550 117819735
675858157 -946995531
532189616 -471831624
-758644861 520356663
298613666 -460909392
616920123 -893494637
-493626381 676624392
251626755 157883871
279171739 797253928
-491591428 -854114883
888504601 -738115609
-181476322 -166289569
423587642 -357757886
-876648407 37534353
-771241187 -278328807
-5315040 60335344
-496548128 326902726
-244376073 -124288893
-910072613 27981992
313856874 727422383
-700928069 -440777193
783274182 60892226
684799157 773659046
-488391085 -98669832
708770388 43975432
-679390985 -690553236
510836170 996517008
-395383253 -592026441
294532949 -542971772
525808958 -141817275
9375182 -102389948
-885796341 -898110965
-695946862 -412433549
-139133289 -701440755
514462458 691550
969167755 -573443940
-98496139 -161841601
388169084 -555679040
-33771574 -82876168
510836170 996517008
952482398 200790065
-977517822 -503747814
861123009 617656977
-275053491 301431470
898192260 -118932270
-685414936 -526412541
363223957 246772408
-755867530 342920794
-647217971 385739068
791227431 -715143252
523364685 -380595373
674682811 673190533
-624497474 -35581801
-167861958 499521574
236790033 448385677
-51075479 965516638
-34824795 -842788423
492361410 -873832178
575061710 -256767814
985359845 -260588255
196565994 134880221
620499894 639622247
670761658 -876284894
-616940511 739507770
-595583642 354821135
-644734340 -332830982
-665882282 -956185790
577413203 -777377866
192846099 -412052440
736985242 -725783711
74895158 363213972
703965569 932079637
-56372163 978153741
765832481 -842401816
521950292 384935203
-471457320 -68093102
265905631 -828146609
-403147178 68880460
-51069971 180009223
55981438 950955857
-729827642 -69182659
-493626381 676624392
-885021854 -100251748
904439756 -779408733
736853163 -97676370
-987758389 887363068
-581382064 440530559
-475324781 -969205878
185771114 411664738
635145413 -644370314
-970078100 444121410
-943142440 -786954143
-703840468 60603512
-51069971 180009223
-709700539 769633766
-51075479 965516638
-534908200 -290248042
