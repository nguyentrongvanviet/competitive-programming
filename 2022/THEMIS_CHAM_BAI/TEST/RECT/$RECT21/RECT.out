493039573808509085